NuRadioMC/examples/Sensitivities/E2_fluxes3.py

```diff
@@ -303,7 +303,10 @@ def get_E2_limit_figure(diffuse=True,
                         show_grand_200k=False,
                         show_radar=False,
                         show_Heinze=True,
-                        show_TA=False):
+                        show_TA=False,
+                        show_RNOG=False,
+                        show_IceCubeGen2=False,
+                        shower_Auger=True):
 
     # Limit E2 Plot
```

Comment:
Not really a comment to the code: But does this belong in this PR?

Reply:
I was working on this branch all the time. It might have slipped into it (or maybe it was even a separte PR that was merged into this branch)